### PR TITLE
Doc introduction changes

### PR DESCRIPTION
**Pull request by mDuo13**
Rewrite of the introductory content to make the docs much more approachable with less context and foreknowledge.

### CODE REVIEW

File: docs/src/introduction.md.ejs [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Introduction
 
-RippleAPI allows you to query and submit transactions to a node on the Ripple network.
+RippleAPI is the official client library to the Ripple Consensus Ledger. Currently, RippleAPI is only available in JavaScript. 
+Using RippleAPI, you can:
+
+* [Query transactions from the network](#gettransaction)
+* [Sign](#sign) transactions securely without connecting to any server
+* [Submit](#submit) transactions to the Ripple Consensus Ledger, including [Payments](#payment), [Orders](#order), [Settings changes](#settings), and [other types](#transaction-types)
+* [Generate a new Ripple Address](#generateaddress)
```
Comment (clark800) [inline docs/src/introduction.md.ejs]: We should say "and more" or something so that it doesn't sound like this is all that it can do.

**Comment by clark800** [comment]
The tests are failing because we added a rule that the `docs/index.md` file must always be up-to-date. You can regenerate it with `npm run docgen`.

**Comment by mDuo13** [comment]
Ah, that explains it. I'll fix & push as soon as I get all the node modules installed.

**Comment by clark800** [comment]
LGTM

**Pull request merged**